test/e2e/performanceprofile/functests/8_performance_workloadhints/workloadhints.go

```diff
@@ -90,22 +90,24 @@ var _ = Describe("[rfe_id:49062][workloadHints] Telco friendly workload specific
 		})
 		When("workloadHint RealTime is disabled", func() {
 			It("should update kernel arguments and tuned accordingly to realTime Hint enabled by default", func() {
+				currentWorkloadHints := profile.Spec.WorkloadHints
 				By("Modifying profile")
 				profile.Spec.WorkloadHints = nil
 
 				profile.Spec.RealTimeKernel = &performancev2.RealTimeKernel{
 					Enabled: pointer.Bool(false),
 				}
+				// If current workload hints already contains the changes skip updating
+				if !(cmp.Equal(currentWorkloadHints, profile.Spec.WorkloadHints)) {
```

Comment:
if the profile is the same, will it trigger MCP update anyway?

Reply:
No it won't trigger